### PR TITLE
Change testnet and mainnnet contract address in missions app

### PR DESCRIPTION
**Pull request by Ishagoyal**

### CODE REVIEW

File: src/common/constants.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const NetworkContextName = 'NETWORK';
 
 export const testnetContractAddress = '0x0735c7eb714a1d67559a3acbee637462223f9f4dfa5d45cbe142c06e57255d1c';
```
Comment (vnaysngh-mudrex) [inline src/common/constants.js:13]: are we not using this testnet contract?

Comment (vnaysngh-mudrex) [reply src/common/constants.js:13]: we shall remove it

File: src/config.js
Comment (vnaysngh-mudrex) [inline src/config.js:1]: are we using contracts from both of these files?

File: src/common/constants.js (hunk 1 above)
Comment (Ishagoyal) [reply src/common/constants.js:13]: We are using it.

File: src/config.js
Comment (vnaysngh-mudrex) [reply src/config.js:1]: we should only use config file

Comment (Ishagoyal) [reply src/config.js:1]: I know, I kept it for improvisations later, else testing area will increase

Comment (vnaysngh-mudrex) [reply src/config.js:1]: okay

**Review by vnaysngh-mudrex — APPROVED** [review]

**Pull request merged**